app/component/no-favourites-panel.scss

```diff
@@ -0,0 +1,35 @@
+ /* No favourites image*/
+.nofavs-img {
+  background: $no-favourites-img;
+  background-size: 100% auto;
+  background-repeat: no-repeat;
+  height: 160px;
+  width: 100%;
+  max-width: 160px;
+  margin: 0 auto;
+  margin-bottom: 1em;
+  margin-top: 2.5em;
+}
+
+@media (max-width: 900px) {
+  .nofavs-img {
+    height: 240px;
+    width: 100%;
+    max-width: 240px;
+  }
+}
+
+@media (max-width: 380px) {
```

Comment:
Is there an explicit reason to start using media queries in steps instead of the .bp-s and .bp-l classes for major breakpoint and just scale the content within .bp-s? If there is, should we move to using this practice everywhere?